include/zarray/zarray.hpp

```diff
@@ -94,6 +94,8 @@ namespace xt
         const nlohmann::json& get_metadata() const;
         void set_metadata(const nlohmann::json& metadata);
 
+        zarray_impl* make_view(xstrided_slice_vector& slices);
```

Comment:
THis method should return a `zarray` object wrapping the `zarray_impl*` returned by the `make_view` method of `zarray_impl`